src/main/java/com/depromeet/domain/feed/application/FeedService.java

```diff
@@ -0,0 +1,96 @@
+package com.depromeet.domain.feed.application;
+
+import com.depromeet.domain.feed.dto.response.FeedOneByProfileResponse;
+import com.depromeet.domain.feed.dto.response.FeedOneResponse;
+import com.depromeet.domain.follow.dao.MemberRelationRepository;
+import com.depromeet.domain.follow.domain.MemberRelation;
+import com.depromeet.domain.member.domain.Member;
+import com.depromeet.domain.mission.domain.MissionVisibility;
+import com.depromeet.domain.missionRecord.dao.MissionRecordRepository;
+import com.depromeet.domain.missionRecord.domain.MissionRecord;
+import com.depromeet.global.util.MemberUtil;
+import com.depromeet.global.util.SecurityUtil;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+// TODO: Redis 사용해서 캐싱 작업 필요
+@Service
+@RequiredArgsConstructor
+@Transactional
+public class FeedService {
+    private final MemberUtil memberUtil;
+    private final MissionRecordRepository missionRecordRepository;
+    private final MemberRelationRepository memberRelationRepository;
+    private final SecurityUtil securityUtil;
+
+    @Transactional(readOnly = true)
+    public List<FeedOneResponse> findAllFeed() {
+        final Member currentMember = memberUtil.getCurrentMember();
+        List<Member> members =
+                memberRelationRepository.findAllBySourceId(currentMember.getId()).stream()
+                        .map(MemberRelation::getTarget)
+                        .collect(Collectors.toList());
+        members.add(currentMember);
+
+        return missionRecordRepository.findFeedAll(members);
+    }
+
+    @Transactional(readOnly = true)
+    public List<FeedOneByProfileResponse> findAllFeedByTargetId(Long targetId) {
+        final Long sourceId = securityUtil.getCurrentMemberId();
+
+        if (!isMyFeedRequired(targetId, sourceId)) {
```

Comment:
```suggestion
        if (isMyFeedRequired(targetId, sourceId)) {
```
로 수정하고 순서 바꾸는게 더 좋을 것 같아요